cdap-ui/app/cdap/components/CaskWizards/PluginArtifactUpload/UploadJarStep/index.js

```diff
@@ -41,12 +41,9 @@ const mapDispatchWithDNDFileProps = (dispatch) => {
     },
   };
 };
-const ArtifactUploader = connect(
-  mapStateWithDNDFileProps,
-  mapDispatchWithDNDFileProps
-)(FileDnD);
+const ArtifactUploader = connect(mapStateWithDNDFileProps, mapDispatchWithDNDFileProps)(FileDnD);
 
-export default function UploadJarStep(undefined, context) {
+export default function UploadJarStep(_, context) {
```

Comment:
I think it would be better to just call the `_` as props. `_` is already a global variable convention for lodash.

Reply:
🤦‍♂ didn't realize that. Will change this.